test/integration/smoke/test_vm_extra_config_data.py

```diff
@@ -0,0 +1,299 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" BVT tests for Virtual Machine additional configuration
+"""
+# Import Local Modules
+from marvin.cloudstackTestCase import cloudstackTestCase
+from marvin.cloudstackAPI import (updateVirtualMachine,
+                                  deployVirtualMachine,
+                                  destroyVirtualMachine,
+                                  stopVirtualMachine,
+                                  startVirtualMachine)
+from marvin.lib.utils import *
+
+from marvin.lib.base import (Account,
+                             ServiceOffering,
+                             VirtualMachine,
+                             Host,
+                             Iso,
+                             Router,
+                             Configurations)
+from marvin.lib.common import (get_domain,
+                               get_zone,
+                               get_template,
+                               list_hosts,
+                               get_test_template)
+from nose.plugins.attrib import attr
+
+# Import System modules
+import urllib
+import xml.etree.ElementTree as ET
+from lxml import etree
```

Comment:
I don't think this is part of stock marvin installation, it will fail when we try to run smoketests against it.

Reply:
They should be part of the python standard library, I do not remember if I had to pip install them but I think they come as default.

Reply:
not in my setup, I have a python with just Marvin and it's dependencies and PyCharm ide cannot resolve lxml

Reply:
Is the a workaround?

Reply:
I've just checked with a marvin installation and it's there, looks like it's missing on Mac, or we add it during the setup. Let leave this for now